lib/charms/traefik_k8s/v2/ingress.py

```diff
@@ -98,6 +99,7 @@ def _on_ingress_revoked(self, event: IngressPerAppRevokedEvent):
         "name": {"type": "string"},
         "strip-prefix": {"type": "string"},
         "redirect-https": {"type": "string"},
+        "scheme": {"type": "string"},
```

Comment:
Iiuc, adding "scheme" shouldn't require us to bump to v2.

Reply:
my reasoning was: ingress v1 is effectively deprecated once we merge this one, so why should we backport new features to it?
Or is that deeply wrong @simskij ?

Reply:
pending a final choice on this one @simskij, we're going ahead as-is. If we want to backport scheme support to v1, that'll be a separate task.